interface/src/app/sample_annotation/annotation.tpl.html

```diff
@@ -0,0 +1,27 @@
+<h1>ADAGE: Sample Annotations</h1>
+
+<h3 class="text-warning" ng-bind="ctrl.queryStatus"></h3>
+
+<table ng-hide="ctrl.queryStatus"
+       class="table table-bordered table-striped table-hover table-condensed">
+  <thead>
+    <th ng-if="ctrl.hasNode">Activity</th>
+    <th>Sample Name</th>
+    <th ng-repeat="typeName in ctrl.uniqueAnnotationTypes">
+      {{typeName}}
+    </th>
+  </thead>
+
+  <tbody>
+    <tr ng-repeat=
+        "sample in ctrl.samples | orderBy: ctrl.hasNode ? '-activity' : 'name'">
```

Comment:
see above note about changing from `ctrl.hasNode` to `ctrl.nodeInUrl`, if you make use of the block of code I rewrote.